src/client/components/Navbar.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+function Navbar() {
+  return (
+    <nav className="Navbar">
+      <div className="PagesNavLink">
+        <ul>
+          {/* <Link to="/about">
+            <li>About</li>
+          </Link> */}
+          <Link to="/meals">
+            <li>Our Meals</li>
+          </Link>
+          <Link to="/add">
+            <li>Share Meal</li>
+          </Link>
+        </ul>
+      </div>
+      <div className="Logo">
+        <Link to="/">
+          <img
+            src="src\client\assets\images\meal-share-logo-bg.png"
```

Comment:
URLs use forward slashes but it might not fix the issue with rendering this image. Perhaps check out the link provided in the class Slack channel to see if that helps.